proto/wallet/wallet.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package proto.wallet;
+option go_package = "github.com/textileio/go-auctions-client/gen/wallet;wallet";
+
+message ProposalSigningRequest {
+	string auth_token = 1;
+
+	string filecoin_deal_protocol = 2;
```

Comment:
It's very unlikely we need to make changes to the message itself, but maybe not a bad idea to version it so, for example, when a new field is added to the protocol, new clients can check if the message has a valid new field or not by checking the version, instead of testing the field value itself, which is not reliable enough.

A message version can cover deal proposal format changes too I think.

Reply:
If we change the `ProposalSigningRequest` or response I think is easier to just make a new version of the [protocol version](https://github.com/textileio/go-auctions-client/pull/1/files#diff-6dc4054cf1607c7c1e1415ba59f68670659a308d79e333c71429d86d7d8b0b2dR18). We can leverage that existing versioning for messages.

Reply:
Ahh right I forgot it, that's better. So dealerd can simply handle two streams to support both old and new signers, if that ever happens.